R/readGT3X.R

```diff
@@ -82,6 +95,9 @@ read.gt3x <- function(path, verbose = FALSE, asDataFrame = FALSE,
   path = unzip_zipped_gt3x(path, cleanup = cleanup)
   remove_path = path
   remove_file = attr(file, "remove")
```

Comment:
What is the "file" object here? It has not been defined so this is always NULL?

Reply:
> Thanks for this update. A small note is that I would prefer that assingment was always done with "<-". Please clarify the use of attr(file, "remove").

I tend to ascribe to `=`.  Can you put in `lintr` code to do this?  The ` attr(file, "remove")` is used later to remove the file from the temporary directory for cleanup as these files can get big if extracted.

Reply:
The object 'file' hasn't been decleared. Therefore 'file' refers to the base R function called file() and attr(file, "remove") is always NULL. Is there something I am missing?

Reply:
It should be `path`.